README.md

```diff
@@ -0,0 +1,16 @@
+# homebrew-socktainer
+
+Homebrew tap which allows installing [socktainer/socktainer](https://github.com/socktainer/socktainer) using the [Homebrew](https://brew.sh/) package manager.
+
+## Installation
+
+```shell
+brew tap socktainer/socktainer
+brew install socktainer
```

Comment:
do you think we should rename this repository to homebrew-tap

so it'll be `brew tap socktainer/tap` (avoid to have twice the socktainer)

Reply:
Personally I do think so, for example I used AWS as a reference when setting up my personal tap:
https://github.com/aws/homebrew-tap